api/controllers/v1/jobs.ts

```diff
@@ -177,6 +181,9 @@ async function NotifyBuildSummary(jobId: string): Promise<void> {
     ),
     entitlement['slack_user_id']
   );
+  return {
```

Comment:
why are we returning status codes from these now? is that required? adds typing errors below (ln 260) from Promise<void> to Promise<any>